### PR TITLE
Wanko

### PR DESCRIPTION
**Pull request by wanko**
Add local bib-entries for SI Algorithms publications

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -606,6 +638,18 @@ @article{aldoma18a
   year = {2018}
 }
 
+@article{aldoma18b,
+  title = {Nurse (re) scheduling via answer set programming},
```
Comment (tortinator) [inline krr.bib:642]:
```suggestion
  title = {Nurse (re)scheduling via answer set programming},
```

File: krr.bib [hunk 2]
```diff
@@ -2184,6 +2236,17 @@ @inproceedings{bamcso00a
...
 }
 
+@inproceedings{bamuru04a,
+  title = {A New Approach to Modeling and Solving Minimal Perturbation Problems},
+  author = {R. Bart{\'a}k and T. M{\"u}ller and H. Rudov{\'a}},
+  editor = {K. Apt and F. Fages and F. Rossi and P. Szeredi and J. V{\'a}ncza},
+  booktitle = {Recent Advances in Constraints},
+  doi = {10.1007/978-3-540-24662-6_13},
+  pages = {233-249},
+  publisher = {Springer Berlin Heidelberg},
```
Comment (tortinator) [inline krr.bib:2246]:
```suggestion
  publisher = springer,
```

File: krr.bib [hunk 3]
```diff
@@ -2866,6 +2939,21 @@ @inproceedings{benisc99a
   year = {1999}
 }
 
+@book{benmic06a,
+  title = {Networks on chips --- technology and tools},
+  author = {L. Benini and G. {De Micheli}},
+  publisher = {Elsevier Morgan Kaufmann},
```
Comment (tortinator) [inline krr.bib:2945]: @string?

File: krr.bib [hunk 4]
```diff
@@ -8746,6 +8843,18 @@ @article{elgesc22a
...
 }
 
+@article{elgesc22a,
+  title = {Problem Decomposition and Multi-shot ASP Solving for Job-shop Scheduling},
+  author = {M. El-Kholany and M. Gebser and K. Schekothin},
+  doi = {10.48550/arXiv.2205.07537},
+  journal = tplp,
+  number = {4},
+  pages = {623-639},
+  publisher = {Cambridge University Press},
```
Comment (tortinator) [inline krr.bib:8853]: @string?

File: krr.bib [hunk 5]
```diff
@@ -14783,6 +14903,15 @@ @inproceedings{khmashja03a
   pages = {272-286}
 }
 
+@inproceedings{khrosa16a,
+  title = {A modular design space exploration framework for multiprocessor real-time systems},
+  author = {N. Khalilzad and K. Rosvall and I. Sander},
+  booktitle = {In Proceedings of Forum on Specification and Design Languages (FDL'16)},
```
Comment (tortinator) [inline krr.bib:14909]:
```suggestion
  booktitle = {Proceedings of Forum on Specification and Design Languages (FDL'16)},
```

File: krr.bib [hunk 6]
```diff
@@ -16034,13 +16163,16 @@ @article{lierler14a
   year = {2014}
 }
 
-@article{lierler21a,
-  title = {Constraint Answer Set Programming: Integrational and Translational (or {SMT}-based) Approaches},
+@article{lierler23a,
+  title = {Constraint Answer Set Programming: Integrational and Translational
+(or {SMT}-based) Approaches},
```
Comment (tortinator) [inline krr.bib:16168]:
```suggestion
  title = {Constraint Answer Set Programming: Integrational and Translational (or {SMT}-based) Approaches},
```

File: krr.bib [hunk 7]
```diff
@@ -16034,13 +16163,16 @@ @article{lierler14a
...
   author = {Y. Lierler},
-  journal = {CoRR},
-  url = {http://arxiv.org/abs/2107.08252},
-  volume = {abs/2107.08252},
-  year = {2021}
+  doi = {10.1017/S1471068421000478},
+  journal = {Theory and Practice of Logic Programming},
```
Comment (tortinator) [inline krr.bib:16171]:
```suggestion
  journal = tplp,
```

File: krr.bib [hunk 8]
```diff
@@ -17000,6 +17132,15 @@ @inproceedings{luglha08a
   year = {2008}
 }
 
+@inproceedings{luglhate08a,
+  title = {Efficient symbolic multi-objective design space exploration},
+  author = {M. Lukasiewycz and M. Gla{\ss} and C. Haubelt and J. Teich},
+  booktitle = {Proceedings of the 13th Asia and South Pacific Design Automation Conference (ASP-DAC'08),},
```
Comment (tortinator) [inline krr.bib:17138]:
```suggestion
  booktitle = {Proceedings of the thirteenth Asia and South Pacific Design Automation Conference (ASP-DAC'08),},
```

File: krr.bib [hunk 9]
```diff
@@ -18236,6 +18377,16 @@ @article{mugrae94a
   year = {1994}
 }
 
+@inproceedings{muneha21a,
+  title = {Exploiting Similarity in Evolutionary Product Design for Improved Design Space Exploration},
+  author = {L. M{\"u}ller and K. Neubauer and C. Haubelt},
+  booktitle = {Embedded Computer Systems: Architectures, Modeling, and Simulation},
+  doi = {10.1007/978-3-031-04580-6_3},
+  pages = {33-49},
+  publisher = {Springer International Publishing},
```
Comment (tortinator) [inline krr.bib:18386]:
```suggestion
  publisher = springer,
```

File: krr.bib [hunk 10]
```diff
@@ -18411,6 +18571,15 @@ @inproceedings{nehawasc18b
   year = {2018}
 }
 
+@inproceedings{nehawasc18c,
+  title = {Systematic Test Case Instance Generation for the Assessment of System-level Design Space Exploration Approaches},
+  author = {K. Neubauer and C. Haubelt and P. Wanko and T. Schaub},
+  booktitle = {Workshop Methoden Und Beschreibungssprachen Zur Modellierung Und Verifikation Von Schaltungen Und Systemen},
```
Comment (tortinator) [inline krr.bib:18577]:
```suggestion
  booktitle = {Workshop Methoden und Beschreibungssprachen zur Modellierung und Verifikation von Schaltungen und Systemen},
```

File: krr.bib [hunk 11]
```diff
@@ -23604,6 +23781,17 @@ @article{thmemc85
   year = {1985}
 }
 
+@article{thopim13a,
+  title = {Exploiting domain knowledge in system-level MPSoC design space exploration},
+  author = {M. Thompson and A. D. Pimentel},
```
Comment (tortinator) [inline krr.bib:23786]:
```suggestion
  author = {M. Thompson and A. Pimentel},
```

**Review by tortinator — CHANGES_REQUESTED** [review]
Almost done :nerd_face:

**Comment by tortinator** [comment]
Hi @wanko , there's still work for you :innocent:

File: krr.bib [hunk 12]
```diff
@@ -606,6 +638,18 @@ @article{aldoma18a
...
 }
 
+@article{aldoma18b,
+  title = {Nurse (re) scheduling via answer set programming},
+  author = {M. Alviano and C. Dodaro and M. Maratea},
+  doi = {10.3233/IA-170030},
+  journal = {Intelligenza Artificiale},
+  number = {2},
+  pages = {109-124},
+  publisher = {IOS Press},
```
[the remark this answers is not in the document]
Comment (wanko) [reply krr.bib:648]: done

File: krr.bib (hunk 3 above)
Comment (wanko) [reply krr.bib:2945]: Only occurred once, so I left it.

File: krr.bib (hunk 4 above)
Comment (wanko) [reply krr.bib:8853]: done

File: krr.bib [hunk 13]
```diff
@@ -16034,13 +16165,15 @@ @article{lierler14a
...
-  journal = {CoRR},
-  url = {http://arxiv.org/abs/2107.08252},
-  volume = {abs/2107.08252},
-  year = {2021}
+  doi = {10.1017/S1471068421000478},
+  journal = tplp,
+  number = {1},
+  pages = {195-225},
+  volume = {23},
+  year = {2023}
```
Comment (tortinator) [inline krr.bib:16176]: Are you sure, they are the same ...? One may contain more details, eg appendices than the other ...

File: krr.bib [hunk 14]
```diff
@@ -17000,6 +17133,15 @@ @inproceedings{luglha08a
   year = {2008}
 }
 
+@inproceedings{luglhate08a,
+  title = {Efficient symbolic multi-objective design space exploration},
+  author = {M. Lukasiewycz and M. Gla{\ss} and C. Haubelt and J. Teich},
+  booktitle = {Proceedings of the thirteenth Asia and South Pacific Design Automation Conference (ASP-DAC'08),},
```
Comment (tortinator) [inline krr.bib:17139]:
```suggestion
  booktitle = {Proceedings of the Thirteenth Asia and South Pacific Design Automation Conference (ASP-DAC'08),},
```

File: krr.bib [hunk 15]
```diff
@@ -18236,6 +18378,16 @@ @article{mugrae94a
   year = {1994}
 }
 
+@inproceedings{muneha21a,
+  title = {Exploiting Similarity in Evolutionary Product Design for Improved Design Space Exploration},
+  author = {L. M{\"u}ller and K. Neubauer and C. Haubelt},
+  booktitle = {Embedded Computer Systems: Architectures, Modeling, and Simulation},
```
Comment (tortinator) [inline krr.bib:18384]: Proceedings of ...?

File: krr.bib [hunk 16]
```diff
@@ -18395,6 +18547,15 @@ @inproceedings{nehagl16a
   year = {2016}
 }
 
+@inproceedings{nehagl16a,
+  title = {Supporting composition in symbolic system synthesis},
+  author = {K. Neubauer and C. Haubelt and M. Gla{\ss}},
+  booktitle = {Proceedings of International Conference on Embedded Computer Systems: Architectures, Modeling and Simulation (SAMOS'16)},
```
Comment (tortinator) [inline krr.bib:18553]:
```suggestion
  booktitle = {Proceedings of the International Conference on Embedded Computer Systems: Architectures, Modeling and Simulation (SAMOS'16)},
```

File: krr.bib [hunk 17]
```diff
@@ -18411,6 +18572,15 @@ @inproceedings{nehawasc18b
   year = {2018}
 }
 
+@inproceedings{nehawasc18c,
+  title = {Systematic Test Case Instance Generation for the Assessment of System-level Design Space Exploration Approaches},
+  author = {K. Neubauer and C. Haubelt and P. Wanko and T. Schaub},
+  booktitle = {Workshop Methoden und Beschreibungssprachen zur Modellierung und Verifikation von Schaltungen und Systemen},
```
Comment (tortinator) [inline krr.bib:18578]: Proceedings

File: krr.bib [hunk 18]
```diff
@@ -21941,6 +22111,14 @@ @inproceedings{scluhate06a
   year = {2006}
 }
 
+@inproceedings{scluhate06a,
+  title = {Improving system level design space exploration by incorporating SAT-solvers into multi-objective evolutionary algorithms},
+  author = {T. Schlichter and M. Lukasiewycz and C. Haubelt and J. Teich},
+  booktitle = {Proceedings of IEEE Computer Society Annual Symposium on Emerging VLSI Technologies and Architectures (ISVLSI'06)},
```
Comment (tortinator) [inline krr.bib:22117]:
```suggestion
  booktitle = {Proceedings of the IEEE Computer Society Annual Symposium on Emerging VLSI Technologies and Architectures (ISVLSI'06)},
```

File: krr.bib [hunk 19]
```diff
@@ -23604,6 +23782,17 @@ @article{thmemc85
   year = {1985}
 }
 
+@article{thopim13a,
+  title = {Exploiting domain knowledge in system-level MPSoC design space exploration},
```
Comment (tortinator) [inline krr.bib:23786]:
```suggestion
  title = {Exploiting domain knowledge in system-level {MPSoC} design space exploration},
```

File: krr.bib [hunk 20]
```diff
@@ -25246,6 +25435,17 @@ @inproceedings{zhutru13a
   pages = {556-568}
 }
 
+@article{zithlafofo03a,
+  title = {Performance assessment of multiobjective optimizers: an analysis and review},
+  author = {E. Zitzler and L. Thiele and M. Laumanns and C.M. Fonseca and V.G. da Fonseca},
```
Comment (tortinator) [inline krr.bib:25440]:
```suggestion
  author = {E. Zitzler and L. Thiele and M. Laumanns and C. Fonseca and V. da Fonseca},
```

**Review by tortinator — CHANGES_REQUESTED** [review]
Almost there. @wanko :nerd_face:

**Comment by wanko** [comment]
@tortinator hopefully all good now :)

**Review by tortinator — APPROVED** [review]

**Comment by tortinator** [comment]
Hi @wanko , I approved but CI failed :(

**Comment by wanko** [comment]
> Hi @wanko , I approved but CI failed :(

I don't understand, I've run the script again and pushed but it still fails. Will have a closer look tomorrow.

**Comment by wanko** [comment]
I did the same thing again in a fresh conda environment and now it worked. I love that we have non-deterministic technology, that makes our automata so much shorter. :D

**Pull request merged**